src/frequenz/sdk/microgrid/component/_component.py

```diff
@@ -67,16 +74,32 @@ class ComponentCategory(Enum):
     """Possible types of microgrid component."""
 
     NONE = components_pb.ComponentCategory.COMPONENT_CATEGORY_UNSPECIFIED
+    """Unspecified component category."""
+
     GRID = components_pb.ComponentCategory.COMPONENT_CATEGORY_GRID
+    """Grid component."""
+
     METER = components_pb.ComponentCategory.COMPONENT_CATEGORY_METER
+    """Meter component."""
+
     INVERTER = components_pb.ComponentCategory.COMPONENT_CATEGORY_INVERTER
+    """Inverter component."""
+
     BATTERY = components_pb.ComponentCategory.COMPONENT_CATEGORY_BATTERY
+    """Battery component."""
+
     EV_CHARGER = components_pb.ComponentCategory.COMPONENT_CATEGORY_EV_CHARGER
+    """EV charger component."""
+
     CHP = components_pb.ComponentCategory.COMPONENT_CATEGORY_CHP
+    """CHP component."""
 
-    # types not yet supported by the API but which can be inferred
-    # from available graph info
     PV_ARRAY = 1000001
+    """PV array component.
```

Comment:
I wonder if in the documentation we should expand acronyms like EV, CHP, PV, or if we should rely on the glossary...

Opinions @matthias-wende-frequenz @shsms?

Reply:
No harm in describing them in detail here, but these are low level interfaces, and we don't expect people who aren't familiar with these components to come and mess around here.  

But maybe we can mention the full forms in the high level interfaces in the logical meter, for `chp*_power` and `pv*_power`